### PR TITLE
Added support for multiple attachments and refactored attachment component

### PR DESCRIPTION
**Pull request by 4manasa**

### CODE REVIEW

**Review by paulgagnon-pega — APPROVED** [review]

File: packages/react-sdk-components/src/components/widget/Attachment/Attachment.tsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-import { CircularProgress } from "@material-ui/core";
-import download from "downloadjs";
-import { getComponentFromMap } from '../../../bridge/helpers/sdk_component_map';
-
+import { CircularProgress } from '@material-ui/core';
+import download from 'downloadjs';
+import { IconButton, Menu, MenuItem } from '@material-ui/core';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import { Button } from '@material-ui/core';
+import { Utils } from '../../helpers/utils';
```
Comment (tumms2021389) [inline packages/react-sdk-components/src/components/widget/Attachment/Attachment.tsx:11]: can you please order these imports in correct order

Comment (4manasa) [reply packages/react-sdk-components/src/components/widget/Attachment/Attachment.tsx]: Done

**Review by tumms2021389 — APPROVED** [review]
Looks good to me

**Comment by tumms2021389** [comment]
@4manasa , Please create a Story make similar changes in Angular SDK as well

**Pull request merged**